msmbuilder/msm/validation/bootstrapmsm.py

```diff
@@ -0,0 +1,135 @@
+#!/bin/env python
+from __future__ import absolute_import, division
+from multiprocessing import Pool, cpu_count
+from msmbuilder.utils import list_of_1d
+from sklearn.utils import resample
+from ..core import _MappingTransformMixin
+from sklearn.base import clone
+from ..msm import MarkovStateModel
+import numpy as np
+
+class BootStrapMarkovStateModel(_MappingTransformMixin):
+    """Bootstrap MSM class which estimates a distribution
+    of transition matrices using random sampling with replacement
+    over the set of input trajectories. The model fits
+    the mle over the original set of sequences as well.
+
+    Parameters
+    ----------
+    n_samples : number of samples to obtain
+    n_procs : number of processes to use=
+
+    Attributes
+    ----------
+    all_populations_ : list of all population vectors
+    resample_ind_ : list of resample indices used to fit each
+    bootstrap mdl. This can be used to regenerate any mdl without
+    having to store it inside this object.
+    mapped_populations_: array (n_samples, mle.n_states_)
+    mapped_populations_mean_:array of mean of population
+    mapped_populations_std_:array of std of population
+    mapped_populations_sem_:array of sem(std/n_samples)
+     of population
+
+    Notes
+    -----
+    The correct number of bootstrap sample is subject to
+    debate with several hundred samples(n_samples)
+    being the recommended starting figure.
+    """
+    def __init__(self, n_samples=10,  n_procs=None, **kwargs):
+        self.n_samples = n_samples
+        self.n_procs = n_procs
+        self.mle = MarkovStateModel(**kwargs)
+
+        self.all_populations_ = None
+        self.mapped_populations_ = None
+        self.resample_ind_ = None
+
+    def fit(self, sequences, y=None):
```

Comment:
Wouldn't n_procs=1 serve the same purpose? I am tempted to default to multiprocessing so that this fits in a somewhat reasonable time for larger datasets.